README.md

```diff
@@ -7,29 +7,28 @@
 [![Release Version](https://img.shields.io/github/v/release/numaproj/numaflow?label=numaflow)](https://github.com/numaproj/numaflow/releases/latest)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6078/badge)](https://bestpractices.coreinfrastructure.org/projects/6078)
 
-## Summary
+Welcome to Numaflow! A Kubernetes-native, serverless platform for running scalable and reliable event-driven applications. Numaflow decouples event sources and sinks from the processing logic, allowing each component to independently auto-scale based on demand. With out-of-the-box sources and sinks, and built-in observability, developers can focus on their processing logic without worrying about event consumption, writing boilerplate code, or operational complexities. Each step of the pipeline can be written in any programming language, offering unparalleled flexibility in using the best programming language for each step and ease of using the languages you are most familiar with.
 
-Numaflow is a Kubernetes-native tool for running massively parallel stream processing. A Numaflow Pipeline is implemented
-as a Kubernetes custom resource and consists of one or more source, data processing, and sink vertices.
+Numaflow, created by the Intuit Argo team to address community needs for continuous event processing, leverages their expertise to deliver a scalable and robust, serverless platform for event-driven applications.
 
-Numaflow installs in a few minutes and is easier and cheaper to use for simple data processing applications than a
-full-featured stream processing platforms.
+![Numaflow Pipeline](./docs/assets/simple-pipeline.png)
```

Comment:
do we need this? it is an oversimplified diagram.

Reply:
Updated the image with a better pipeline